cmd/configschema/go.mod

```diff
@@ -613,30 +613,30 @@ require (
-	go.opentelemetry.io/collector/confmap v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/connector v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/consumer v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/exporter/loggingexporter v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/exporter/otlpexporter v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/extension v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/extension/auth v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/extension/ballastextension v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/extension/zpagesextension v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/featuregate v1.0.0-rcv0014.0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/pdata v1.0.0-rcv0014.0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/processor v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/processor/batchprocessor v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
-	go.opentelemetry.io/collector/semconv v0.84.1-0.20230908201109-ab3d6c5b6470 // indirect
+	go.opentelemetry.io/collector/config/configauth v0.85.0 // indirect
+	go.opentelemetry.io/collector/config/configcompression v0.85.0 // indirect
+	go.opentelemetry.io/collector/config/configgrpc v0.85.0 // indirect
+	go.opentelemetry.io/collector/config/confighttp v0.85.0 // indirect
+	go.opentelemetry.io/collector/config/confignet v0.85.0 // indirect
+	go.opentelemetry.io/collector/config/configopaque v0.85.0 // indirect
+	go.opentelemetry.io/collector/config/configtelemetry v0.85.0 // indirect
+	go.opentelemetry.io/collector/config/internal v0.85.0 // indirect
+	go.opentelemetry.io/collector/confmap v0.85.0 // indirect
+	go.opentelemetry.io/collector/connector v0.85.0 // indirect
+	go.opentelemetry.io/collector/consumer v0.85.0 // indirect
+	go.opentelemetry.io/collector/exporter/loggingexporter v0.85.0 // indirect
+	go.opentelemetry.io/collector/exporter/otlpexporter v0.85.0 // indirect
+	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.85.0 // indirect
+	go.opentelemetry.io/collector/extension v0.85.0 // indirect
+	go.opentelemetry.io/collector/extension/auth v0.85.0 // indirect
+	go.opentelemetry.io/collector/extension/ballastextension v0.85.0 // indirect
+	go.opentelemetry.io/collector/extension/zpagesextension v0.85.0 // indirect
+	go.opentelemetry.io/collector/featuregate v1.0.0-rcv0014 // indirect
```

Comment:
Is this pulling back to the latest release instead of latest snapshot?

Reply:
Not sure I understand what you mean by snapshot. This is preparing the release of contrib by updating all the deps to the latest released version in core

Reply:
The former version was `go.opentelemetry.io/collector/featuregate v1.0.0-rcv0014.0.20230908201109-ab3d6c5b6470` (snapshot indicates September 8th) and now is back to `go.opentelemetry.io/collector/featuregate v1.0.0-rcv0014` which was released last July. Maybe that doesn't matter, but it's happening for pdata as well, which has this change: https://github.com/open-telemetry/opentelemetry-collector/commit/ebed8dd0943a98447d7f0e265cf58ec197bff93a#diff-1f8826a44b89e0c16a541d21b89589fb86cdbbb6fcdccc976cc6a0e96d0834f7

Maybe that overall doesn't matter? Not sure.

Reply:
v1.0.0-rcv0014 is the latest featuregate/pdata version. We got v1.0.0-rcv0014.0.20230908201109-ab3d6c5b6470 pointing to the latest commit in the core repo when `make update-otel` runs without a module. So yes, technically, we are rolling back, but since there are no changes in featuregate or pdata, it's ok